lib/dealership.rb

```diff
@@ -0,0 +1,54 @@
+class Dealership
+  attr_reader :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+    @has_inventory = false
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def has_inventory?
+    @has_inventory
+  end
+
+  def cars_by_make(make)
+    @inventory.select do |car|
+      car.make == make
+    end
+  end
+
+  def total_value
+    total_v = []
+    @inventory.each do |car|
```

Comment:
Nice job solving this with `.each` and then using `.reduce` to return an integer. Take a look at the `.sum` enumerable. 
https://ruby-doc.org/core-2.4.1/Array.html#:~:text=sum-,(,-init%3D0)%20%E2%86%92%20number